src/app/DownloadsDialog.qml

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015-2016 Canonical Ltd.
+ * Copyright 2021 UBports Foundation
```

Comment:
Whoops... In this case, both copyright statements would be kept. The code was under copyright by Canonical, and now we've added some stake with our changes.

Reply:
No, that file was created in #415 
i actually noticed that there are a few others that we missed in that PR :sweat_smile: 
I'm planning to correct them in a separate PR but I just corrected this one here since I modified it anyway.

But this is definitely a lesson learned for me moving forward - don't just copy paste copyright texts :grinning: